ipynb/tests/Frequency_Invariance_Test.ipynb

```diff
@@ -0,0 +1,827 @@
+    }
+   ],
+   "source": [
+    "import logging\n",
+    "from conf import LisaLogging\n",
+    "LisaLogging.setup()\n",
+    "logging.getLogger('Analysis').setLevel(logging.ERROR)\n",
+    "logging.getLogger('Trace').setLevel(logging.ERROR)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run test workload\n",
+    "\n",
+    "There's currently no way to pass a `TestEnv` or configuration to automated test classes. Instead the target information comes from the `target.config` file (in the root of the LISA source tree), so you'll need to edit that to configure LISA to connect to your target."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "    Goal\n",
+      "    ====\n",
+      "    Basic check for frequency invariant load tracking\n",
+      "\n",
+      "    Detailed Description\n",
+      "    ====================\n",
+      "    This test runs the same workload on the most capable CPU on the system at a\n",
+      "    cross section of available frequencies. The trace is then examined to find\n",
+      "    the average activation length of the workload, which is combined with the\n",
+      "    known period to estimate an expected mean value for util_avg for each\n",
+      "    frequency. The util_avg value is extracted from scheduler trace events and\n",
+      "    its mean is compared with the expected value (ignoring the first 300ms so\n",
+      "    that the signal can stabilize). The test fails if the observed mean is\n",
+      "    beyond a certain error margin from the expected one. load_avg is then\n",
+      "    similarly compared with the expected util_avg mean, under the assumption\n",
+      "    that load_avg should equal util_avg when system load is light.\n",
+      "\n",
+      "    Expected Behaviour\n",
+      "    ==================\n",
+      "    Load tracking signals are scaled so that the workload results in roughly the\n",
+      "    same util & load values regardless of frequency.\n",
+      "    \n"
+     ]
+    }
+   ],
+   "source": [
+    "from tests.eas.load_tracking import FreqInvarianceTest\n",
+    "\n",
+    "t = FreqInvarianceTest()\n",
+    "print t.__doc__"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To run automated tests from within a notebook we instantiate the test class and call `init` on it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2017-01-18 18:34:40,898 INFO    : LisaTest     : Setup tests execution engine...\n",
+      "2017-01-18 18:34:40,899 INFO    : TestEnv      : Using base path: /home/bjackman/sources/lisa\n",
```

Comment:
With the restructure of the example notebooks we've discussed it might be better to stick to a generic "/home/vagrant/", but yes, not all notebooks are consistent in that regard at the moment.